### ISSUE TITLE
Slower and bigger model after int8 quantization

### ISSUE THREAD

**Issue by johanforslund** [tensorflow/tensorflow#47760]
@tensorflow/micro

**Describe the problem**
I'm running the person_detection example in tensorflow/lite/micro.

When comparing the old uint8 and the new int8 models, the int8 model requires more space as it's using both SOFTMAX and RESHAPE ops. It also seems to be slower than the old uint8 model.

What is the advantage of using the new int8 model?

**Comment by Saduf2019** [comment]
@johanforslund 
This depends on the model and its implementation, the new models reduce latency and memory usage by hardware.

Kindly open a stackoverflow issue for this as it is not a bug or feature request, Please post this kind of support questions at [Stackoverflow](https://stackoverflow.com/questions/tagged/tensorflow). There is a big community to support and learn from your questions. 
Thanks!

**Comment by napoleonwar** [comment]
@johanforslund Did you run the helloworld and micro speech from micro example before? Can I ask what the code size it is? like the size of binary file, .hex file....
